.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
+# . "$(dirname "$0")/_/husky.sh"
```

Comment:
Why we commented this piece of code?

Reply:
This wasn't supposed to be sent with this commit, will resolve it